src/Tracker.js

```diff
@@ -111,7 +111,7 @@ class Tracker extends React.Component {
             case 'Amber Tablet':
                 this.state.logic.toggleDungeonCompleted('Earth Temple');
                 break;
-            case 'Goddess Harp':
+            case 'Goddess\'s Harp':
```

Comment:
This and the surrounding cases need to be updated for the new location names that are coming with the item name changes.

Reply:
Done. I've done a quick check for references to the old location names and haven't found any others